src/server/main_service.cc

```diff
@@ -1367,6 +1381,11 @@ bool Service::InvokeCmd(const CommandId* cid, CmdArgList tail_args, ConnectionCo
     return false;
   }
 
+  auto reason = cntx->reply_builder()->ConsumeLastError();
```

Comment:
auto -> string

Reply:
?

Reply:
I was investigating a bug. Give me a second I will push this change -- I am not ignoring you

Reply:
done

Reply:
also I am very curious why your IDE can't deduce auto :stuck_out_tongue: